process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessDefinitionKey.java

```diff
@@ -19,12 +19,13 @@
 import com.ritense.processdocument.domain.ProcessDefinitionKey;
 import javax.persistence.Column;
 import javax.persistence.Embeddable;
+import java.io.Serializable;
 import java.util.Objects;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentLength;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentNotNull;
 
 @Embeddable
-public class CamundaProcessDefinitionKey implements ProcessDefinitionKey {
+public class CamundaProcessDefinitionKey implements ProcessDefinitionKey, Serializable {
```

Comment:
Is `Serializable` need here?

Reply:
Yes, the JPA spec required Id's to be serializable. This key is part of one.